react/components/FollowButton/index.js

```diff
@@ -66,12 +73,14 @@ class FollowButton extends Component {
       ...rest
     } = this.props;
 
+    const isFollowed = !!(followable && followable.is_followed);
+
     return (
       <span onClick={this.toggleFollow} role="button" tabIndex={0} {...rest}>
         {{
-          true: 'Unfollow',
-          false: 'Follow',
-        }[followable.is_followed]}
+          false: followNode,
+          true: unfollowNode,
+        }[isFollowed]}
```

Comment:
So this could just be {children} with a default prop of a component that displays Follow/Unfollow, which would then let you customize it accordingly.

Reply:
Ah got it, makes sense.